skpro/distributions/__init__.py

```diff
@@ -3,7 +3,9 @@
 # copyright: skpro developers, BSD-3-Clause License (see LICENSE file)
 # adapted from sktime
 
-__all__ = ["Laplace", "Normal"]
+__all__ = ["Log-Normal","Empirical", "Laplace", "Normal"]
```

Comment:
what state of the repository are you branching off from? This does not look like the most recent version, it looks like it is half a year old. Please make sure you update your fork regularly.